EIPS/eip-6909.md

```diff
@@ -0,0 +1,600 @@
+---
+eip: 6909
+title: Multi-Token
```

Comment:
I think you'll need to expand the title a bit. Multi-token doesn't give the reader enough information about the proposal.

Reply:
expanded to "Minimal Multi-Token Interface"